config/locales/simple_form.en.yml

```diff
@@ -126,6 +126,7 @@ en:
         expires_in: Expire after
         fields: Profile metadata
         header: Header
+        honeypot: Website (do not fill in)
```

Comment:
I suggest adding a honeypot string with placeholder something like `%{label} (do not fill in)` to avoid duplication -- speaking of which, a bit concerned that "do not fill in" might be understood by some bots.

Reply:
They might. But the idea is to strike a balance between blocking bots and not blocking legitimate users. And so far, this seems to have been effective.

Reply:
Refactored.